### PR TITLE
11 set up auth0 login 

### PR DESCRIPTION
**Pull request by akselsf**
Auth0 login works. Stores access token and user in localstorage (change later?)

### CODE REVIEW

File: src/components/authentication/Authcallback.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        catch (e) {
+            console.error(e);
+        }
+    }
+
+    useEffect(() => {
+     
+        
+        storeUser();
+    }, [isAuthenticated, user]);
```
Comment (julian-ao) [inline src/components/authentication/Authcallback.tsx:29]:
```suggestion
    useEffect(() => {
        storeUser();
    }, [isAuthenticated, user]);
```

**Comment by akselsf** [comment]
Navbar/dropdown needs redesign but has functionality

File: src/components/universal/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Stjålet rett fra opptaksiden shoutout
```
Comment (julian-ao) [inline src/components/universal/Button.tsx:1]: shoutout beste opptaksiden

File: src/components/universal/Navbar.tsx [hunk 3]
```diff
@@ -2,56 +2,150 @@ import React, { useState } from 'react';
...
+  given_name?: string;
+}
+
+type NavdropdownProps = {
+  user?: User;
+  logout: () => void;
+  login: () => void;
+  isAuthenticated: boolean;
+
+}
```
Comment (julian-ao) [inline src/components/universal/Navbar.tsx:31]:
```suggestion
  isAuthenticated: boolean;
}
```

File: src/components/universal/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  href?: string;
+}
+
+const Button = (props: Props) => {
+  let colorClasses = "";
+  let sizeClasses = "";
+
+  
+
+  if (props.size === "small") {
```
Comment (julian-ao) [inline src/components/universal/Button.tsx:18]:
```suggestion
  let sizeClasses = "";

  if (props.size === "small") {
```

**Pull request merged**